include/seqan3/argument_parser/detail/format_html.hpp

```diff
@@ -182,18 +182,16 @@ class format_html : public format_help_base<format_html>
         // Print legal stuff
         if ((!meta.short_copyright.empty()) || (!meta.long_copyright.empty()) || (!meta.citation.empty()))
         {
-            std::cout << "<h2>Legal</h2>\n<strong>";
+            std::cout << "<h2>Legal</h2>\n";
```

Comment:
Why are you deleting the strong here without replacing it with a in_bold?

Reply:
It is replaced but in the line below. 
Previously the person who wrote/copied this (probably me :see_no_evil:) tried to save some ... strings I guess.
E.g. 
```
std::cout << "foo<strong>"
          << var << "</strong>"; // var is bold
```
instead of 
```
std::cout << "foo"
          << "<strong>" << var << "</strong>"; // var is bold
```